web/instances/instances.go

```diff
@@ -374,11 +375,16 @@ func cleanOrphanAccounts(c echo.Context) error {
 			args = fmt.Sprintf("0 %d %d * * %d", m, h, d)
 		}
 
+		md, err := metadata.NewWithApp(acc.AccountType, "", job.DocTypeVersionTrigger)
```

Comment:
I wouldn't use `acc.AccountType` for `createdByApp`, as it is the stack that creates the trigger, not the konnector